src/adapter/src/coord/statement_logging.rs

```diff
@@ -506,6 +508,11 @@ impl Coordinator {
             execution_timestamp: None,
             application_name: session.application_name().to_string(),
             transaction_isolation: session.vars().transaction_isolation().to_string(),
+            transaction_id: session
+                .transaction()
+                .inner()
+                .expect("Every statement runs in an explicit or implicit transaction")
```

Comment:
this `.expect(...)` scares me a bit. What do you think about making the column nullable?

Reply:
Is it actually possible for the transaction to be None at this point? If so, why? @jkosh44 and @mjibson you guys might also know

Reply:
If a statement is being executed, it's in some kind of transaction. This is correct. `Default` is what happens when there's not an explicit transaction, or while pgwire is waiting for new commands to execute.